packer/instance-scripts/init-quorum.sh

```diff
@@ -274,6 +274,27 @@ function wait_for_terraform_provisioners {
     done
 }
 
+function associate_elastic_ip {
+    load readonly AWS_REGION=$1
```

Comment:
I've never seen `load` used in bash. What does it do and are you sure you didn't mean `local`?

Reply:
You got me, must've misread earlier in the file -- I'm just trying to patternmatch you